front-end/package.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
i get a number of high severity warnings installing here. while some minor warnings are ok, is there anything we can do to get rid of these warnings?